test/docs/__snapshots__/get-docs-yaml.api-test.ts.snap

```diff
@@ -200,6 +229,40 @@ components:
         - citizenshipClass
         - officerRiskDate
         - countryCode
+    PostPartyRequestItem:
+      type: object
+      properties:
+        alternateIdentifier:
+          type: string
+        industryClassification:
+          type: string
+        name1:
+          type: string
+        name2:
+          type: string
+        name3:
+          type: string
+        smeType:
+          type: string
+        citizenshipClass:
+          type: string
+        officerRiskDate:
+          type: string
```

Comment:
This should be a date

Reply:
It's not a JS/TS date though, just a string representation of a date. I can use `DateString`, but as that's just an alias for string that seems to make no difference to the generated yaml

Reply:
You need to tell the @ApiProperty decorator that it should be different: I think you should specify these options in the decorator:
```
    type: Date,
    format: 'date',
```

Reply:
`format: 'date'` gets added to the swagger docs but `type: Date` has no effect - is it possible that the _true_ type of the property take precedence over that specified in @ApiProperty? Is `format: 'date'` sufficient?